README.md

```diff
@@ -160,8 +160,8 @@ $ node -e "console.log('Hello from Node.js ' + process.version)"
 Prerequisites:
 
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
-* Visual Studio 2013 for Windows Desktop, or
-* Visual Studio Express 2013 for Windows Desktop
+* Visual Studio 2013 / 2015 for Windows Desktop, or
```

Comment:
The full edition doesn't have a "for Windows Desktop" SKU. Only the Visual Studio Express does and it was called "for Windows Desktop" in 2013 and "for Desktop" in 2015.